README.md

```diff
@@ -176,6 +178,7 @@ var opt = {
 | `config`      | String / Object | _String_ : A URL string from which to load a [Vega](https://vega.github.io/vega/docs/config/)/[Vega-Lite](https://vega.github.io/vega-lite/docs/config.html) or [Vega-Lite](https://vega.github.io/vega-lite/docs/config.html) configuration file. This URL will be subject to standard browser security restrictions. Typically this URL will point to a file on the same host and port number as the web page itself. <br> _Object_ : A Vega/Vega-Lite configuration as a parsed JSON object to override the default configuration options. |
 | `theme`       | String        | If specified, tells Vega-Embed use the theme from [Vega Themes](https://github.com/vega/vega-themes). **Experimental: we may update themes with minor version updates of Vega-Embed.** |
 | `defaultStyle` | Boolean or String | If set to `true` (default), the embed actions are shown in a menu. Set to `false` to use simple links. Provide a string to set the style sheet. |
+| `bindContainer` | String or Element | The element that should contain any input elements bound to signals. |
```

Comment:
In vega, when it's passed in as an object it's called bind, not bindContainer. It's called bindContainer when passed in as a positional argument (where it doesn't matter). We should try to be consistent, but I'm okay with either of course :)

Reply:
Good catch. Fixed.